collection-mocks.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @summary The mock collection variable. A null collection used to mock the results of a db operation
+ */
+var mockCollection = new Mongo.Collection(null);
+
+/**
+ * @summary The mock method. It is applied to the Mongo.Collection prototype and mocks the results of a db operation
+ * @locus Anywhere
+ * @method mock
+ * @memberOf Mongo.Collection
+ * @param {Object} insert An array of arguments to be passed to the mockCollection's insert method
+ * @param {Object} update An array of arguments to be passed to the mockCollection's update method
+ * @param {Object} find An array of arguments to be passed to the mockCollection's findOne method
+ * @return {Object} The result of findOne
+ *
+ * XXX Should the params be documented as MongoSelector instead?
```

Comment:
The arguments to this function are actually quite complex, but we could certainly be more specific about what they are. For now `Object|Array` would probably be the best description.